serverless.yml

```diff
@@ -1,5 +1,5 @@
 service:
-  name: StackJanitor
+  name: StackJanitor${self:custom.NAME_SUFFIX}
```

Comment:
I just got a bit confuse why we need the name_suffix here. Is that for your testing ?

Reply:
Yep - just so I could deploy multiple stack janitors in one account (they'd have the same CFn stack name without this suffix). Happy to keep it here as it's useful for future testing